src/main/java/io/kestra/plugin/templates/Example.java

```diff
@@ -19,6 +20,20 @@
     title = "Short description for this task",
     description = "Full description of this task"
 )
+@Plugin(
+    examples = {
+        @io.kestra.core.models.annotations.Example(
+            full = true,
```

Comment:
Full is usualy used for a complete flow.
Best to only add the task properties as it's the kind of example we usually add